deploy/README.md

```diff
@@ -161,16 +161,30 @@ kubectl -n sumologic create secret generic sumologic \
   --from-literal=endpoint-events=$ENDPOINT_EVENTS
 ```
 
-Apply `fluentd-sumologic.yaml` manifest with following command:
+If you don't need to customize the configuration apply the `fluentd-sumologic.yaml` manifest with the following command:
 
 ```sh
 curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
 sed 's/\$NAMESPACE'"/sumologic/g" | \
 kubectl -n sumologic apply -f -
 ```
 
+If you need to customize the configuration there are two commands to run. First, get the `fluentd-sumologic.yaml` manifest with following command:
+
+```sh
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
+sed 's/\$NAMESPACE'"/sumologic/g" >> fluentd-sumologic.yaml
+```
+
+You can customize the provided YAML file if needed, including configuring fields for [events](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/fluent-plugin-events/README.md#fluent-plugin-events). Once done run the following command to apply the `fluentd-sumologic.yaml` manifest.
```

Comment:
nit: Since these are the steps under `If you need to customize the configuration...`, I feel we can reword the `You can customize the provided YAML file if needed` (since otherwise the user would have run the previous command under `If you don't need to customize...`)